less/index.less

```diff
@@ -53,7 +53,188 @@ table {
     box-sizing: border-box;
 }
 
+font {
+	font-family: 'Special Elite', cursive;
+	font-size: 62.5%;
+}
+
 html, body {
-    height: 100%;
+	height: 100%;
+}
+
+h1 {
+	font-size: 3.8rem;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+// ======================================= variables
+@text-color: #4C4F56;
+@button-color: #8C7561;
+@divider-color: #ECEDEE;
+@check-border: 0.1px solid red;
+@bkgrd-color: #D8C5A5;
+
+// ======================================= mixins
+.center-all() {
```

Comment:
OK, now I am seeing the issue. This mixin is the root of your troubles. I'll say it again, vw is dangerous at any size, but especially at 100. I would definitely not use it in mixin, because it seems like you lost track of that and applied it in spaces that you didn't mean to.  Make sure you double check what your mixins are doing when you start adding them. Separating mixins and importing them will make them easier to find. Check your mixin naming conventions and make sure their function is obvious.